.github/workflows/image_base.yml

```diff
@@ -32,10 +37,14 @@ jobs:
           username: ${{ secrets.username }}
           password: ${{ secrets.password }}
       - name: Build and push a base image for building Kepler with libbpf
+        id: build-push-image
         uses: docker/build-push-action@v6
         with:
           context: .
           file: ./build/Dockerfile.builder
           platforms: linux/amd64,linux/arm64,linux/s390x
           push: ${{ inputs.pushImage }}
-          tags: quay.io/sustainable_computing_io/kepler_builder:ubi-9-libbpf-1.3.0
+          tags: quay.io/rh-ee-asavage/kepler_builder:ubi-9-libbpf-1.3.0
```

Comment:
I think we need to revert this line back to:
```
tags: quay.io/sustainable_computing_io/kepler_builder:ubi-9-libbpf-1.3.0
```

Before merge

Reply:
Change added and rebased.